aodntools/timeseries_products/Documentation/velocity_aggregated_timeseries.md

```diff
@@ -0,0 +1,124 @@
+from ONE site
+
+optional arguments:
+  -h, --help                    show this help message and exit
+  -site SITE_CODE               site code, like NRMMAI
+  -files FILENAMES              name of the file that contains the source URLs
+  -path OUTPUT_PATH             path where the result file will be written. Default: ./
+  -indir INPUT_DIR              base path of input files
+  -outdir OUTPUT_DIR            path where the result file will be written. Default ./
+  -download_url DOWNLOAD_URL    path to the download_url_prefix
+  -opendap_url OPENDAP_URL      path to the opendap_url_prefix
+
+
+```
+
+
+
+### Input file validation
+
+Before proceeding to the aggregation, each input file will be checked to ensure it meets the requirements (as specified above under Inputs). Any input files that fail to meet the requirements will be excluded from the aggregation, and their URL listed in a global attribute `rejected_files`.
+
+### Dimensions
+
+The dimensions of the resulting file  are determined as follows:
+
+- `OBSERVATION`:    the total number of observation records, excluding out-of-the-water data, in all input files;
+- `INSTRUMENT`:     the number of instruments (i.e. number of files);
+- `string`:         a string dimension for character array variables.
+
+### Variables
+
+The velocity variables are produced by flattening, then concatenating the arrays in each of the input files. The resulting variable has dimension `OBSERVATION`. Each variable’s ancillary_variables, in particular the corresponding quality-control flags, are also included, with dimension `OBSERVATION`. If the quality control variable is absent in any input file, the corresponding flags in the output file will  be set to 0 (“no QC performed”).
+
+The variable `TIME` from input files is re-shaped to match the flattened velocity variables, then concatenated into a variable `TIME(OBSERVATION)`. This will result in a non-uniform time interval and repeated timestamps.
+
+The `DEPTH` variable from input files is concatenated into a variable `DEPTH(OBSERVATION)`. In the case of ADCP instruments, the `HEIGH_ABOVE_SENSOR`  is converted to absolute depth by subtracting each of the height values from the depth measurements at the instrument. `DEPTH_quality_control`, if present, is also included. 
+
+All output variables with the `INSTRUMENT` dimension are sorted in chronological order, and the input files aggregated chronologically, according to the global attribute time_deployment_start.
+
+In order to keep track of the provenance of the aggregated file, accessory variables are created:
+
+- `instrument_index(OBSERVATION)`: index [0:number of files] of the instrument used, referencing the `INSTRUMENT` dimension.
+- `source_file(INSTRUMENT, string)`: URLs of the files used
+- `instrument_id(INSTRUMENT, string)`: concatenated deployment_code, instrument and instrument_serial_number from the global attributes of each file
```

Comment:
Again, remove `string` dimension here, unless we end up including it.

Reply:
removed